WorkInProgress/Administration/xASL_adm_CoregisterAtlas.m

```diff
@@ -122,12 +191,38 @@
 spm_jobman('run',matlabbatch);
 
 % Create MAT file
-IM = xASL_io_Nifti2Im(fullfile(pathTPM, 'oAAL3v1_1mm.nii'));
-IM = uint8(IM);
-save(fullfile(pathTPM, 'oAAL3v1_1mm.nii.mat'),'IM');
-xASL_adm_GzipNifti(fullfile(pathTPM, 'oAAL3v1_1mm.nii'));
-
 IM = xASL_io_Nifti2Im(fullfile(pathTPM, 'oWMPM_Type_III.nii'));
-IM = uint8(IM);
+IM = uint8(ceil(IM/2.0)); % Make sure it's divided by 2 and rounded to deal with the L-R and merge them to bilateral
```

Comment:
This doesn't work for each atlas, most have left and right but also single bilateral regions; so this needs checking per ROI atlas!

Reply:
Yes. Fully agree!!! But for these two cases, it works right?

Reply:
Fix with the correct script.